### ISSUE TITLE
Can't install plugin with logstash release >=8.5.1

### ISSUE THREAD

**Issue by maederm** [cherweg/logstash-input-s3-sns-sqs#77]
Hi

I can't install the plugin with logstash >=8.5.1. It just hangs during the `bin/logstash-plugin install` command.

I can reproduce it in logstash releases 8.5.1, 8.5.2, 8.5.3 and 8.6.0.

Here is an example docker-compose.yml file to see the issue.

```yaml
version: '2.3'
services:
  ls860:
    image: docker.elastic.co/logstash/logstash:8.6.0
    command: bin/logstash-plugin install logstash-input-s3-sns-sqs
    networks:
      - elastic

networks:
  elastic:
    driver: bridge
```

Cheers, maederm

**Comment by rteterin** [comment]
Same issue here. When running `logstash-plugin` with `DEBUG=1`, it hangs forever on `Resolving dependencies...`

**Comment by metalshanked** [comment]
same issue here. any fix or workaround? thx!

**Comment by DesAWSume** [comment]
Also looking for the v8 working version.

**Comment by camilodiazsal** [comment]
I managed to install the plugin by building the gem file first, and then installs successfully.

**Comment by lucascott** [comment]
+1 on this issue, would be nice to have the linked fix pushed forward

**Comment by saocean** [comment]
@cherweg ,

Is there a plan to address this? It looks like @camilodiazsal has a workaround.

Thanks

**Comment by DesAWSume** [comment]
Can someone have a look into this PR and move this forward?

**Comment by christianherweg0807** [comment]
should be fixed with release 2.2.2
